### PR TITLE
[SW-560] refactor(middleware): remove unnecessary workarounds

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

File: src/lib/wallet.js [hunk 1]
```diff
@@ -48,18 +48,6 @@ async function initMiddleware() {
...
-          in: 'path',
-          name: 'id',
-          required: true,
-          type: 'string',
-        }],
-      },
-    },
   };
   spec.basePath = '/mdw/';
   // TODO: remove after resolving https://github.com/aeternity/ae_mdw/issues/160
```
Comment (subhod-i) [inline src/lib/wallet.js:53]: Something to remove here too.

Comment (CedrikNikita) [reply src/lib/wallet.js:53]: Their behaviour did not change from the middleware side, so, not yet.

Comment (peronczyk) [reply src/lib/wallet.js:53]: I confirm what Nikita said - I tested this within my recent work.

**Comment by Liubov-crypto** [comment]
LGTM

**Review by peronczyk — APPROVED** [review]

**Review by subhod-i — APPROVED** [review]

**Pull request merged**